### DISCUSSION TITLE
How  to pass a binary file data to python

### DISCUSSION THREAD

**Discussion by bestneymarjr** [pyscript/pyscript#2164] [Q&A] [answered]
I have a sqlite db in web worker. I want to export the database to binary file data, then pass the data to python.After doing this, I can use Python to analyze database data. Can you tell me how to pass data to python？Thanks!

**Comment by WebReflection** [comment, marked as answer]
If you are able to extract the DB as *Uint8Array* you can create a *Blob* then a URL that will serve that blob and you can fetch it via pyscript fetch as `array_buffer()` which will give you a memoryview of that data ... you can then pass it to sqlite (if that's available in pyodide) as db to use and the rest is regular SQL.

That being said, it's not clear what is your current stack so any extra detail might help.

**Reply by bestneymarjr** [reply]
Thank you very much, I will try it.

**Reply by bestneymarjr** [reply]
it's worked, Thanks again!
